src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -136,7 +136,7 @@ public static void cancel(BukkitTask task)
 
     public static boolean isExecutive(String name)
     {
-        return ConfigEntry.SERVER_OWNERS.getStringList().contains(name) || ConfigEntry.SERVER_EXECUTIVES.getStringList().contains(name);
+        return ConfigEntry.SERVER_OWNERS.getStringList().contains(name) || ConfigEntry.SERVER_EXECUTIVES.getStringList().contains(name) || ConfigEntry.SERVER_ASSISTANT_EXECUTIVES.getStringList().contains(name);
```

Comment:
Have you confirmed there are no permissions execs have that we wouldn't want assistants not to have?